util.js

```diff
@@ -1,9 +1,29 @@
-const parseJson = (jsonRaw, jsonError) => {
+import { isResponse, isSuccess} from './phapi';
+
+export const parseJson = (jsonRaw, jsonError) => {
     try {
         return JSON.parse(jsonRaw);
     } catch(exception) {
         return jsonError;
     }
 }
 
-export default parseJson;
\ No newline at end of file
+export const buildResponse = (phResponse, ...objs) => {
+
+    if (isResponse(phResponse)) {
+      var result = { result: phResponse.result };   
+      if (isSuccess(phResponse)) {
+        for (var obj of objs) {
```

Comment:
replace the for-loop with `Object.assign(result, ...objs)`

Reply:
fixed